dist/gitlabci/RoboFile.php

```diff
@@ -207,4 +174,19 @@ protected function runComposer() {
     return $tasks;
   }
 
+  /**
+   * Install Drupal.
+   *
+   * @return \Robo\Task\Base\Exec
+   *   A task to install Drupal.
+   */
+  protected function installDrupal()
```

Comment:
This might be enough for some cases but for BDD (now Behat, Cypress.io in the future), a fresh database might not be enough to test some of the scenarios. This is why I relied upon a database image. Thoughts?

Reply:
Yup, right now behat is only working on TravisCI and CircleCI.

My idea was to unify the approach on Github Actions and GitlabCI to use the `DB_DUMP_URL` and `wget` + DB import approach used on the above two. I'll try to work on that approach next so we have the same kind of tests available for all platforms.